### DISCUSSION TITLE
code does run - module not find

### DISCUSSION THREAD

**Discussion by gbfarias** [HelgeGehring/gdshelpers#34] [General] [answered]
Hi! I've installed the package through pip but it does not run the examples! I get the error message below.
Any thoughts??

![Captura de pantalla 2020-11-13 174115](https://user-images.githubusercontent.com/36306469/99097176-8b49d480-25d7-11eb-9808-2a3488105d96.png)

**Comment by HelgeGehring** [comment, marked as answer]
Hey, it seems as if shapely is not installed correctly. As you're using anaconda, I'd guess installing shapely via conda could help.
So, first uninstall shapely via pip `pip uninstall shapely` and then install it again via conda `conda install shapely`.
Hope that helps!
Helge

**Reply by gbfarias** [reply]
Thanks that works now!
